Directory.Build.props

```diff
@@ -1,7 +1,8 @@
 <Project>
```

Comment:
Tests are executed framework by framework in [ci.yml](https://github.com/dotnet/docfx/blob/ee4ab806de415ec47e0ed382615246e49ab52c3f/.github/workflows/ci.yml#L35) to fail fast, consider run tests against net8.0 there. I make tests against non-mandonary TFMs ubuntu only to make CI fast.

Reply:
Done